app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/CustomPlaybackTransportControlGlue.java

```diff
@@ -59,6 +73,68 @@
         initActions(context);
     }
 
+    @Override
+    protected PlaybackRowPresenter onCreateRowPresenter() {
+        final AbstractDetailsDescriptionPresenter detailsPresenter =
+                new AbstractDetailsDescriptionPresenter() {
+                    @Override
+                    protected void onBindDescription(ViewHolder
+                                                             viewHolder, Object obj) {
+                        PlaybackTransportControlGlue glue = (PlaybackTransportControlGlue) obj;
+                        viewHolder.getTitle().setText(glue.getTitle());
+                        viewHolder.getSubtitle().setText(glue.getSubtitle());
+                    }
+                };
+
+        PlaybackTransportRowPresenter rowPresenter = new PlaybackTransportRowPresenter() {
+            @Override
+            protected RowPresenter.ViewHolder createRowViewHolder(ViewGroup parent) {
+                RowPresenter.ViewHolder vh = super.createRowViewHolder(parent);
+                Context context = getContext();
```

Comment:
```suggestion
                Context context = parent.getContext();
```

That one should always be not-null